earthfile2llb/earthfile2llb.go

```diff
@@ -97,6 +99,16 @@ func Earthfile2LLB(ctx context.Context, target domain.Target, opt ConvertOpt) (m
 	if err != nil {
 		return nil, errors.Wrapf(err, "resolve build context for target %s", target.String())
 	}
+
+	ftrs, err := features.GetFeatures(bc.Earthfile.Version)
+	if err != nil {
+		return nil, errors.Wrapf(err, "resolve build context for target %s", target.String())
```

Comment:
(You'll need to also define a `bc.Earthfile.Version.String()` or similar)

```suggestion
		return nil, errors.Wrapf(err, "resolve feature set for version %s for target %s", bc.Earthfile.Version.String(), target.String())
```

Reply:
Currently the structs in the ast/spec package only have JSON tags and no methods (e.g. see https://github.com/earthly/earthly/blob/version/ast/spec/earthfile.go ). A quick idea is to just use `%v` and `bc.Earthfile.Version.Args`, or we could just marshal the str via `json.marshal`?